en/messages.json

```diff
@@ -298,21 +298,23 @@
         "message": "With Relay Premium, you can block promotional emails from reaching your inbox while still allowing you to receive emails like receipts or shipping information.",
         "description": "DO NOT TRANSLATE \"Relay Premium\"."
     },
-    "popupBundlePromoHeadline": {
+    "popupBundlePromoHeadline_2": {
         "message": "Add Mozilla VPN at $SAVINGS$ off.",
-        "description": "DO NOT TRANSLATE \"Mozilla VPN\"",
+        "description": "DO NOT TRANSLATE \"Mozilla VPN\". This string includes a savings variable, e.g. 50%",
```

Comment:
I added the 'example' context in description for both the altered strings. Is this better? @peiying2